src/consensus/aft/raft.h

```diff
@@ -517,12 +515,28 @@ namespace aft
             }
           }
         }
+
+        if (orc_sets.find(idx) == orc_sets.end())
+        {
+          orc_sets[idx] = {};
+        }
+
+        if (resharing_tracker)
+        {
+          kv::NetworkConfiguration netconfig = {idx, {}}; // TODO: Fix
```

Comment:
The resharing tracker needs to know the exactly list of nodes in the next configuration (the nodes in that config need to agree on a new distribution of key shares and it can't include nodes from earlier or later configurations). Can we still get that by running over `configurations` and match on the `rid`?

Reply:
I believe it is possible but haven't explored this yet as I'm waiting for the CI to run on the current changes first.